packages/shared/src/lib/flow-run/flow-status.ts

```diff
@@ -0,0 +1,5 @@
+import { FlowRunStatus } from './execution/flow-execution'
+
+export const getFlowState = (status: FlowRunStatus): boolean => {
```

Comment:
If this returns a boolean I think it's wrong to call it get

I would rename it to isFinishedFlowState or isTerminalFlowState